src/index.ts

```diff
@@ -83,6 +84,7 @@ export function parseCookies(
   ctx?:
     | Pick<next.NextPageContext, 'req'>
     | { req: next.NextApiRequest }
+    | { req: express.Request }
```

Comment:
Is that truly all that's missing?! That's awesome 😄

Reply:
Yeah, surprisingly!